test/Frontend.test.ts

```diff
@@ -0,0 +1,57 @@
+import { Template, Match } from '@aws-cdk/assertions';
+import { 
+    Stack,
+    App,
+    aws_apigateway as apigw,
+} from 'aws-cdk-lib';
+import FrontEnd from '../lib/FrontEnd';
+
+
+describe('FrontEnd Test', () => {
+
+    const app = new App();
+    const stack = new Stack(app, 'MyTestStack');
+
+    const mockApi = new apigw.RestApi(stack, 'TestApi', {});
+    mockApi.root.addMethod('GET', new apigw.MockIntegration({}));
+
+    new FrontEnd(stack, 'FrontEndTest', {
+        ApiGateway: mockApi
+    });
+
+    const assert = Template.fromJSON(app.synth().getStackArtifact(stack.artifactId).template);
+
+    test('CloudFront distribution count == 1', () => {
+        assert.resourceCountIs('AWS::CloudFront::Distribution', 1);
+    });
+    
+    test('S3 Bucket count == 1', () => {
+        assert.resourceCountIs('AWS::S3::Bucket', 1);
+    });
+    
+    test('CloudFront default behaviour == S3', () => {
```

Comment:
Can we add a test to make sure that the default caching policy is applied to the S3 origin

Reply:
I guess [this](https://github.com/phonemkyaw/Demos.CDKApiPipeline/blob/21ca2e5b459fea1bbb788e2d7e7a850bb5a3de73/test/Frontend.test.ts#L32 ) is the test?

Reply:
That tests that the origin is S3. If you look at the specific properties on it you'll see the cache policy ID of `4135ea2d-6df8-44a3-9df3-4b5a84be39ad` in there - this is the part I'm looking for. Have a look at [the doco for managed cache policies](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-managed-cache-policies.html) and you can see the ID's of the default policies so you know where I got that from.